### ISSUE TITLE
React Native rnpm warning

### ISSUE THREAD

**Issue by driley-emoney** [rollbar/rollbar-react-native#78]
After upgrading to react native 60 it gave this warning:

```
warn The following packages use deprecated "rnpm" config that will stop working from next release:
  - rollbar-react-native: https://npmjs.com/package/rollbar-react-native
Please notify their maintainers about it. You can find more details at https://github.com/react-native-community/cli/blob/master/docs/configuration.md#migration-guide.

```

**Comment by abumostafa** [comment]
+1

**Comment by mattlm** [comment]
+1

**Comment by vorks** [comment]
+1

**Comment by waltjones** [comment]
@driley-emoney Thank you for the report. Fixed in 0.6.1. PR: https://github.com/rollbar/rollbar-react-native/pull/83
